packages/tempus/src/Tempus_StepperBDF2_decl.hpp

```diff
@@ -118,6 +123,7 @@ class StepperBDF2 : virtual public Tempus::StepperImplicit<Scalar>
   Scalar                                     order_;
 
   Teuchos::RCP<Thyra::VectorBase<Scalar> >   xDotTemp_;
+  Teuchos::RCP<const Thyra::VectorBase<Scalar> >      initial_guess_;  
```

Comment:
It looks like several different Steppers repeat the same code.  Why not put this in a common base class?